src/ucp/rma/basic_rma.c

```diff
@@ -60,19 +76,38 @@ ucp_rma_request_advance(ucp_request_t *req, size_t frag_length,
     }
 }
 
+/**
```

Comment:
the comment is relevant for the two function (bcopy/zcopy_completion) and it is also somewhat relevant for the rma_request_adcance. May be I should merge the comment with the one before ucp_rma_request_advance ?